### ISSUE TITLE
invalid syntax on python 3.4.1

### ISSUE THREAD

**Issue by alauener** [tehmaze/ipcalc#38]
Hi 
I am trying to install ipcalc with pip on python version 3.4.1 and get this message:

Installing collected packages: ipcalc
  Running setup.py install for ipcalc
      File "C:\Python34\Lib\site-packages\ipcalc.py", line 237
        if ip > 0xffffffffffffffffffffffffffffffffL:
                                                  ^
    SyntaxError: invalid syntax

Is it possible, that you can fix that?

Kind regard
Adrian

**Comment by kottenator** [comment]
The same error for me

**Comment by tehmaze** [comment]
Cannot reproduce locally and I don't have access to a Windows machine:

    ~$ pip3 install ipcalc
    Collecting ipcalc
      Downloading ipcalc-1.1.3.tar.gz
    Installing collected packages: ipcalc
      Running setup.py install for ipcalc
    Successfully installed ipcalc-1.1.3
     ~$ python3 --version
    Python 3.4.3

Will have to investigate what triggers the issue on Windows / Python 3.4.1

**Comment by nullpass** [comment]
Using a fresh Python3.4.3 install on Windows it does work fine. 

```
c:\Python34\Scripts>pip3 install ipcalc
You are using pip version 6.0.8, however version 6.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Collecting ipcalc
  Downloading ipcalc-1.1.3.tar.gz
Installing collected packages: ipcalc
  Running setup.py install for ipcalc
Successfully installed ipcalc-1.1.3

c:\Python34\Scripts>
```

**Comment by wcrissman** [comment]
Using Python 3.4.3 on OSx Yosemite, `pip install` works but trying to use it in the REPL does not.
```
~> pip3 install ipcalc
Collecting ipcalc
  Downloading ipcalc-1.1.3.tar.gz
Installing collected packages: ipcalc
  Running setup.py install for ipcalc
Successfully installed ipcalc-1.1.3
```
```
~> python3
Python 3.4.3 (default, May  1 2015, 19:14:18)
[GCC 4.2.1 Compatible Apple LLVM 6.1.0 (clang-602.0.49)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import ipcalc
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.4/site-packages/ipcalc.py", line 237
    if ip > 0xffffffffffffffffffffffffffffffffL:
                                              ^
SyntaxError: invalid syntax
>>> exit()
```

**Comment by nullpass** [comment]
I'm sorry, I feel pretty dumb. I could have sworn Python3 compat had already been merged. As of now only Python2 will work. The invalid syntax error is actually related to how int() works in Py3, there is no long type, many number formats were added to int().

**Comment by wcrissman** [comment]
Thanks for the explanation. Do you know when Python 3 support will be added?

**Comment by tehmaze** [comment]
Just merged Python 3.4 support from the milestone-2.x branch into main. Please test this release with:

    $ pip install git+https://github.com/tehmaze/ipcalc
